Gemfile.lock

```diff
@@ -0,0 +1,158 @@
+GIT
```

Comment:
I kept the commit separate so that it can easily be dropped in case we don't want to check it in.